docs/pages/bssw/bssw_styling_common.md

```diff
@@ -62,15 +63,34 @@ The main body is the portion of the article below the deck. BSSw.io uses [Github
 
-- Add the relative path to the image file from the `*.md` file. Using the "logo" class helps constrain the size of the image.
+- Add the relative path to the image file from the `*.md` file. 
 ````
-<img src='../../images/YOUR-IMAGE-NAME.png' class='logo'/>
+<img src='../../images/YOUR-IMAGE-NAME.png' class='logo'/>[optional caption]
 ````
-- Please ensure we have permission to use the logo/image 
+- Please ensure we have permission to use the logo/image
 - Please ensure logo is clear and high resolution
 - Please remember to commit the image file with the `*.md` file on your git branch
+- Use the `logo` class to constrain the rendered size of the image.
+- Use the `page` class to prevent enlargement (recommended for horizontal images under 1000 px wide).
+- Use the `page lightbox` class for vertical images and images that have details that could be better viewed if the image were expanded to fill the screen.
+- Add an image credit as appropriate.
+- Captions are optional (and in general, mildly discouraged) and must immediately follow the img tag if used.
 
-See  [images/README.md](https://github.com/betterscientificsoftware/bssw.io/blob/master/images/README.md) for more details.
+Additional points about images and captions:
+
+- Captions should not include hyperlinks.  They are undesirable from the design standpoint: it disrupts reading the text (your eye jumps straight to the link) and any critical links would be best placed in the story itself.  Further, hyperlinks prevent the front-end from actually interpreting the caption as a caption.
+
+- If we are providing image credits that include links, it seems unnecessary to link to the image resource, especially if not a scientific source, even if the URL is part of the identifier. (For example, we did not to link to nasa.gov on the Apollo series since it is such a vast resource, we credited Image Source: NASA.)
+
+- Blog articles must have *either* a hero image or deck text.
+
+- Hero image captions: Many images are illustrative and self-explanatory or have tangential relevance to the article (e.g. data visualization) so a caption is unnecessary. With images that benefit from descriptions that provide insight into the BSSw activities or the rigorousness of the science we have added a sentence of text. For an example, see <https://github.com/betterscientificsoftware/bssw.io/blob/master/Articles/Blog/2020-11-PSIP4HDF5.md> (source) and <https://bssw.io/blog_posts/recent-successes-with-psip-on-hdf5> (rendered).
+
+- Hero image credits: We've come up with a way to lead the article but not detract from the lede paragraph by leading body text with image credits but shrinking them with a superscript tag. For an example, see <https://github.com/betterscientificsoftware/bssw.io/blob/master/Articles/Blog/2021-03-useful-practices-for-SEoMsDSP.md> (source) and <https://bssw.io/blog_posts/useful-practices-for-software-engineering-on-medium-sized-distributed-scientific-projects> (rendered).
+
```

Comment:
Why use `<sup>...</sup>` instead of just explicitly setting font size with `<p style="font-size:10px">..</p>`. The former is shorter but also moves text position.

Reply:
> <sup>...</sup>

@markcmiller86: where are you referring to when you mention <sup>...</sup>?

Reply:
I am referring to the guidance...

> ...but shrinking them with a superscript tag.

which I believe means to use `<sup>...</sup>` to shrink the text. 

My remark was to suggest that we don't do that and instead use actual style settings to do that.

Reply:
Good point. I am not familiar with the hero image formatting, so will defer this to @bernhold

Reply:
This is what Matt has recommended and what we use (in at least one case that I know of).  I propose that we accept this PR as documenting current practice and open a new ticket about whether we want to continue doing it this way.  There's a lot of other material in the PR, and I've already referred to it several times while working on other articles.

Reply:
I am fine with that.

Reply:
I created issue #1477 for this concern.